js/repl/compile.js

```diff
@@ -101,6 +98,7 @@ export default function compile(code: string, config: CompileConfig): Return {
       //   compiled = prettier.__debug.formatAST(transformed.ast, DEFAULT_PRETTIER_CONFIG);
       // } else {
       compiled = prettier.format(compiled, DEFAULT_PRETTIER_CONFIG);
+      meta.compiledSize = new Blob([compiled], { type: "text/plain" }).size;
```

Comment:
Can't we only do the `meta.compiledSize` calculation once, by just moving it to after the if-prettier check?